src/components/AddBareMetalHosts/OCMClusterDetailTabContent.tsx

```diff
@@ -1,25 +1,50 @@
 import React from 'react';
-import { Modal, ModalVariant } from '@patternfly/react-core';
 import { normalizeClusterVersion } from '../../config';
 import { OcmClusterType } from './types';
 import { AddHostsClusterCreateParams, Cluster, getCluster, handleApiError } from '../../api';
-import RedirectToCluster from '../ui/RedirectToCluster';
 import { getOcmClusterId } from './utils';
 import { usePullSecretFetch } from '../fetching/pullSecret';
 import { ErrorState, LoadingState } from '../ui';
 import { addHostsClusters } from '../../api/addHostsClusters';
+import { AlertsContextProvider } from '../AlertsContextProvider';
+import { POLLING_INTERVAL } from '../../config';
+import AddBareMetalHosts from './AddBareMetalHosts';
 
-const AddBareMetalHostsDialog: React.FC<{
-  isOpen: boolean;
-  closeModal: () => {};
-  cluster: OcmClusterType;
-}> = ({ isOpen, cluster, closeModal }) => {
+type ClusterDetailTabType = {
+  key: number;
+  title: string;
+  contentId: string;
+  id: string;
+  show: boolean;
+  ref: React.RefObject<unknown>;
+};
+
+export const getOCMClusterDetailTab = (
+  tabs: ClusterDetailTabType[],
+  addBareMetalTabRef: React.RefObject<unknown>,
+  displayAddBareMetalHosts: boolean,
+): ClusterDetailTabType => ({
+  key: tabs.reduce((max, tab) => (tab.key > max ? tab.key : max), tabs[0].key) + 1,
+  title: 'Bare Metal',
+  contentId: 'addBareMetalHostsContent',
+  id: 'addBareMetalHosts',
+  show: displayAddBareMetalHosts,
+  ref: addBareMetalTabRef,
+});
+
+export const OCMClusterDetailTabContent: React.FC<{
```

Comment:
This component name is quite confusing when being used on OCM side. I'd prefer calling it 'AddBareMetalHostsClusterDetailTab`

Reply:
Good point, changing to `BareMetalHostsClusterDetailTab`